Jenkinsfile

```diff
@@ -52,7 +52,7 @@ pipeline {
                   stages {
                     stage('Checkout code') { steps { dir('k-exercises') { git url: 'git@github.com:kframework/k-exercises.git' } } }
                     stage('Build and Test K') {
-                      options { timeout(time: 45, unit: 'MINUTES') }
+                      options { timeout(time: 60, unit: 'MINUTES') }
```

Comment:
This step is typically taking less than 25 minutes, so we definitely shouldn't need to increase this timeout @traiansf .

Either there is a systematic slowdown associated with this PR, or some particular test that is causing problems.

Reply:
I saw that by looking on the other PRs. However, since I don't notice something peculiar on my machine, I though that I could try to see whether Jenkins might pass if given enough time.